libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -46,14 +85,27 @@ def __init__(
             connection_string: SQLServer connection string.
             db_schema: The schema in which the vector store will be created.
                 This schema must exist and the user must have permissions to the schema.
+            distance_strategy: The distance strategy to use for comparing embeddings.
+                Default value is COSINE. Available options are:
+                - COSINE
+                - DOT
+                - EUCLIDEAN
             embedding_function: Any embedding function implementing
                 `langchain.embeddings.base.Embeddings` interface.
+            embedding_length: The length (dimension) of the vectors to be stored in the
+                table. Default - None.
+                Note that if `embedding_length` is defined, only vectors of the same
+                size will be accepted. If not defined, vectors of different sizes can
+                be stored in the table, however, similarity search will not be possible
```

Comment:
when we move to use the VECTOR type as the column type, we wont be able to store vectors different sizes. I thought we decided to error out even now. 

Are we still allowing vectors of different sizes to be stored in the table?

Reply:
I wasn't sure if the vector datatype will always require a dimension specified. If that is the case, then we won't be able to store different sizes and I'll need to make changes to this PR.
Just so this is factored in, PG allows storing different sizes.

Reply:
Updated. Confirmed that VECTOR data type requires dimension specified always.